loony_dev/github/client.py

```
@@ -93,6 +93,28 @@ def gh_api(self, endpoint: str) -> list | dict:
             return []
         return json.loads(output)
 
+    def gh_api_patch(self, endpoint: str, **fields: str) -> dict:
+        """PATCH a GitHub REST API endpoint, passing fields as JSON body via stdin."""
+        payload = json.dumps(fields)
+        cmd = ["gh", "api", "-X", "PATCH", f"repos/{self.repo}/{endpoint}", "--input", "-"]
+        max_retries = int(gh_setting("max_retries"))
+        backoff = float(gh_setting("initial_backoff"))
+        for attempt in range(max_retries + 1):
+            try:
+                result = subprocess.run(cmd, input=payload, capture_output=True, text=True, check=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)